README.md

```diff
@@ -23,17 +25,21 @@ Note: We recommend running these commands in a unix terminal. Windows users can
 
 1. Clone the repo.
     ```shell
-    git clone https://github.com/plaid/pattern-account-funding.git
-    cd pattern
+    git clone https://github.com/plaid/pattern-transfers.git
+    cd pattern-transfers
     ```
+1. Follow the instructions for [Transfer UI](https://plaid.com/docs/transfer/using-transfer-ui/#prerequisites-for-using-transfer-ui) to set up your link customization on the [Plaid developer dashboard](https://dashboard.plaid.com/team/api).
+
 1. Create the `.env` file.
     ```shell
     cp .env.template .env
     ```
-1. Update the `.env` file with your [Plaid API keys][plaid-keys] and OAuth redirect uri (in sandbox this is 'http<span>://localhost:3002/oauth-link'</span>).
+1. Update the `.env` file with your [Plaid API keys][plaid-keys] and OAuth redirect uri (in sandbox this is 'http<span>://localhost:3002/oauth-link'</span>) and link customization name.
 
 1. You will also need to configure an allowed redirect URI for your client ID through the [Plaid developer dashboard](https://dashboard.plaid.com/team/api).
 
+1. Finally, contact your Plaid Account Executive or Account Manager to make sure that your clientId is enabled for the Transfer product.
```

Comment:
replace this with the actual business logic we will be using at GA (I assume this app won't be launched before GA) -- everyone has Sandbox access, fill out the form for production or development access